tests/integration/test_clamav_config.py

```diff
@@ -0,0 +1,156 @@
+"""Test ClamAV configuration verification."""
+
+import argparse
+import pathlib
+import tempfile
+from unittest.mock import patch
+
+from nilrt_snac._configs._clamav_config import _ClamAVConfig
+
+
+class TestClamAVConfig:
+    """Test cases for ClamAV configuration verification."""
+
+    def test_configure_not_installed(self):
+        """Test configure method when ClamAV is not installed."""
+        config = _ClamAVConfig()
+
+        # Mock opkg_helper to return False for all packages
+        with patch.object(config._opkg_helper, "is_installed", return_value=False):
+            args = argparse.Namespace(dry_run=False)
+
+            # This should not raise an exception
+            config.configure(args)
+
+    def test_verify_not_installed(self):
+        """Test verify method when ClamAV is not installed - should pass."""
+        config = _ClamAVConfig()
+
+        # Mock opkg_helper to return False for all packages
+        with patch.object(config._opkg_helper, "is_installed", return_value=False):
+            args = argparse.Namespace()
+            result = config.verify(args)
+
+            assert result is True
```

Comment:
In the CLI test where it runs the entire script, it also harvests stdout and verifies specific strings are included. You wouldn't do that (or at least not exactly that) here which is an integration test for just this component. Do you have access to the logger output used by the call to verify? It would be nice to see that this included "skipping verification" in the output, to make sure it didn't return true for the wrong reason.